spec/components/accordion_sections/application_information_component_spec.rb

```diff
@@ -153,12 +154,10 @@
   end
 
   context "when the work has been started" do
-    let(:application_type) { create(:application_type, :ldc_existing) }
-
-    it "renders 'Yes'" do
+    it "renders 'session id'" do
       render_inline(component)
 
-      expect(page).to have_row_for("Work already started:", with: "Yes")
+      expect(page).to have_row_for("Session ID", with: "21161b70-0e29-40e6-9a38-c42f61f25ab9")
```

Comment:
This appears to be changing an existing spec?

Reply:
Ah yep was trying to duplicate this then edit it, good spot!